src/component/decorators.js

```diff
@@ -37,6 +39,23 @@ export const autoName = {
   decorate(component) {
     return {
       ...component,
+      async getLabel(id, labelPath) {
+        const propertyPath = isGraphPointer(labelPath)
+          ? toSparql(labelPath)
```

Comment:
toSparql confuses me, it suggests it returns SPARQL. But does this return a predicate?

Reply:
It returns a SPARQL Property Path

Reply:
So a single predicate (`Predicate Path` in SHACL lingo) will be just a standard property